tools/IoTEdgeDevOps/DevOpsLib/AzureLogAnalytics.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace DevOpsLib
+{
+    using System;
+    using System.Net.Http;
+    using System.Threading.Tasks;
+
+    public sealed class AzureLogAnalytics
+    {
+        static readonly AzureLogAnalytics instance = new AzureLogAnalytics();
+        static readonly string apiVersion = "v1";
+
+        // Explicit static constructor to tell C# compiler
+        // not to mark type as beforefieldinit
+        static AzureLogAnalytics()
+        {
+        }
+
+        AzureLogAnalytics()
+        {
+        }
+
+        public static AzureLogAnalytics Instance
+        {
+            get
+            {
+                return instance;
+            }
+        }
+
+        // Use get-request to do a Kusto query
+        // API reference: https://dev.loganalytics.io/documentation/Using-the-API/RequestFormat
+        public async Task<string> GetKqlQuery(
```

Comment:
```suggestion
        public async Task<string> RunQuery(
```

Reply:
There are multiple ways to run query using Log Analytic endpoint API. 
You can also do `POST` kusto query.  Currently, we are using `GET`; hence, GetKqlQuery()